cmd/template/fallback.cfg.sample

```diff
@@ -0,0 +1,13 @@
+#
+# If you pass this file to nri-resmgr using the --fallback-config
+# command line option, it will be used if configuration cannot be
+# acquired from any other source (agent, or last configuration
+# stored in the cache).
+#
+
+policy:
+  Active: template
+  ReservedResources:
+    CPU: 750m
+logger:
+  Debug: resource-manager,cache,policy,resource-control
```

Comment:
Please add a newline here.

The default fallback file is copied from topology-aware policy and it is probably not applicable for any new policy that this template is based on. Perhaps we should leave this config file from this patch, @klihub WDYT
Currently the fallback is only used for e2e tests so this looks like unnecessary file to have.

Reply:
I think it could be a good idea for demonstrative purposes to add some configuration data to the template policy (if we can't come up with anything else then just some dummy data) and then include a corresponding configuration snippet in the supplied fallback configuration.